test/service/internal_key_event_handlers/outdent_handler_test.dart

```diff
@@ -0,0 +1,268 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import '../../infra/test_editor.dart';
+
+void main() async {
+  setUpAll(() {
+    TestWidgetsFlutterBinding.ensureInitialized();
+  });
+
+  group('outdent_handler.dart', () {
+    testWidgets(
+      "press shift tab in plain text",
+      (tester) async {
+        const text = 'Welcome to Appflowy 😁';
+        final editor = tester.editor
+          ..insertTextNode(text)
+          ..insertTextNode(text)
+          ..insertTextNode(text);
+
+        await editor.startTesting();
+
+        var document = editor.document;
+
+        var selection = Selection.single(path: [0], startOffset: 0);
+        await editor.updateSelection(selection);
```

Comment:
```suggestion
        await editor.updateSelection(Selection.single(path: [0], startOffset: 0));
```
You don't use `var document = editor.document;` - And also it makes no sense when it's already very short.

Reply:
yeah, makes sense.

Reply:
I have made it final, we need to check that the shift+tab doesn't affect the document when pressed with nodes other than list types. Thanks!